src/sql/parts/jobManagement/views/proxiesView.component.ts

```diff
@@ -164,7 +164,11 @@ export class ProxiesViewComponent extends JobManagementView implements OnInit {
 
 	public openCreateProxyDialog() {
 		let ownerUri: string = this._commonService.connectionManagementService.connectionInfo.ownerUri;
-		this._commandService.executeCommand('agent.openProxyDialog', ownerUri);
+		let credentials = [];
+		this.proxies.forEach(proxy => {
```

Comment:
I'm not sure what this list of credentials is for?  It looks like the set of credentials that are referenced by any proxy, which isn't likely to be the full set of credentials available on the system.  Also, I'd prefer if the dialog retrieves it's own metadata so that it is more self-contained (there may be some optimizations if we've already previously queried the required data, but that doesn't appear to be the case here).